Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 0.6.0
-appVersion: v1.9.0
+version: 1.0.0
+appVersion: 1.10.0
```

The `v` is a required part of the tag published to `quay.io/cortexproject/cortex`
```suggestion
appVersion: v1.10.0
```